tests/integration/suite/sentry/metrics/metrics.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/pkg/sentry/server/ca"
+	"github.com/dapr/dapr/tests/integration/framework"
+	procsentry "github.com/dapr/dapr/tests/integration/framework/process/sentry"
+	"github.com/dapr/dapr/tests/integration/framework/util"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(expiry))
+}
+
+// expiry tests the certificate expiry metric.
+type expiry struct {
+	notGiven *procsentry.Sentry
+	given    *procsentry.Sentry
+}
+
+func (e *expiry) Setup(t *testing.T) []framework.Option {
+	rootKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	require.NoError(t, err)
+	onemonth := time.Hour * 24 * 30
+	bundle, err := ca.GenerateBundle(rootKey, "integration.test.dapr.io", time.Second*5, &onemonth)
+	require.NoError(t, err)
+
+	e.notGiven = procsentry.New(t, procsentry.WithDontGiveBundle(true))
+	e.given = procsentry.New(t, procsentry.WithCABundle(bundle))
+
+	return []framework.Option{
+		framework.WithProcesses(e.notGiven, e.given),
+	}
+}
+
+func (e *expiry) Run(t *testing.T, ctx context.Context) {
+	e.notGiven.WaitUntilRunning(t, ctx)
+	e.given.WaitUntilRunning(t, ctx)
+
+	time.Sleep(time.Second)
```

Comment:
This should be removed. Not sure how this got here :grimacing:

Reply:
It's the in release-1.12 branch: https://github.com/dapr/dapr/blob/0b476b18da79fa16cfe52bef0794048f62d2f890/tests/integration/suite/sentry/metrics/metrics.go#L64-L69

If that shouldnt be there please open a PR against release-1.12 to remove it. I will not do it in this merge PR